Rebuild binary.

e2l3n opened the pull request:
<b>Description</b>: Rebuild `GCDWebServer.framework` binary file.

e2l3n commented:
ping @Icenium/core @ligaz @zahhak

mbektchiev commented:
Can you please update the project file to the one that you've used for building the framework?

e2l3n committed:
Automate framework production.

Delete piping to echo command from post-build phase.

Validate that build configuration is 'Release'.

Rebuild GCDWebServer.framework.

Execute script from shell file. Kill xcodebuild process in case of error.

Update submodule from upstream to c98941121a4b96a4fa4ad785790a4a3e119227a5.

Rebuild GCDWebServer.framework with latest update from upstream.

e2l3n commented:
9b31358 adds an Xcode 7.1 project which produces universal static framework with deployment target iOS 6.0. The build phase uses post-build script that produces universal static framework and replaces the existing one.
91831e6 adds newly produced GCDWebServer.framework.

ping @mbektchiev

mbektchiev commented on src/ios/GCDWebServerFramework/GCDWebServer.xcodeproj/project.pbxproj:
```diff
@@ -0,0 +1,484 @@
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "#Don't execute script if CREATE_UNIVERSAL_FRAMEWORK is 0 in order to avoid infinite loops.\nif [ \"${CREATE_UNIVERSAL_FRAMEWORK}\" = 0 ]; then\nexit $?\nfi\n\n#For some reason xcodebuild fails to produce simulator build (i386,x86_64) when one chooses 'Generic iOS device' from Xcode 7.1\n#Xcode error:\n#CodeSign error: entitlements are required for product type 'Framework' in SDK ...
```
I recommend that you extract the script in a `.sh` file and call it from here. Otherwise it'll be much more difficult to read and maintain.

galexandrov commented:
Can we make a build from c98941121a4b96a4fa4ad785790a4a3e119227a5

e2l3n commented:
ping @galexandrov

galexandrov commented:
:+1:

e2l3n commented:
Squash commits.

mbektchiev commented:
:+1:
